packages/build/src/download-center/config.ts

```diff
@@ -103,8 +160,117 @@ export function createVersionConfig(
       os: getDistro(packageVariant),
       name: getDownloadCenterDistroDescription(packageVariant),
       download_link:
-        'https://downloads.mongodb.com/compass/' +
+        publicArtifactBaseUrl +
         getPackageFile(packageVariant, packageInformation).path,
     })),
+  } as const;
+}
+
+interface JsonFeed {
+  versions: JsonFeedVersionEntry[];
+}
+
+interface JsonFeedVersionEntry {
+  version: string;
+  downloads: JsonFeedDownloadEntry[];
+}
+
+interface JsonFeedDownloadEntry {
+  arch: string;
+  distro: string;
+  targets: string[];
+  sharedOpenssl: string | undefined;
+  archive: {
+    type: string;
+    url: string;
+    sha256: string;
+    sha1: string;
+  };
+}
+
+export async function createJsonFeedEntry(
+  packageInformation: PackageInformationProvider,
+  publicArtifactBaseUrl: string
+): Promise<JsonFeedVersionEntry> {
+  const { version } = packageInformation('linux-x64').metadata;
+  return {
+    version: version,
+    downloads: await Promise.all(
+      ALL_PACKAGE_VARIANTS.map(async (packageVariant: PackageVariant) => {
+        const arch = getArch(packageVariant);
+        const distro = getDistro(packageVariant);
```

Comment:
We don't add this distro in the generated json file, just missed out or we can't?

Reply:
There is https://github.com/mongodb-js/mongosh/blob/0b54496fe8024b7982ec1545fedcd02f72e96a3f/packages/build/src/download-center/config.ts#L211 – is that what you mean?

Reply:
Ahh no I mean in here - https://github.com/mongodb-js/mongosh/pull/1562/files#diff-ac13dafadbaacbb5b6de1d253a5ec90b790e2756ca746b2d9bb42177d3cf70b7R106-R115

Where we generate a json feed for existing releases.

Reply:
Oh, yeah. I guess we could try … I’m not sure what the values would be in that case for the pre-1.5.0 releases. I mean, we obviously had some values in the build packages code back then, but I don’t think we could easily apply our _current_ set of `distro` values to those. I’d probably omit it, this can just be a new feature for versions starting now, what do you think?

Reply:
I feel alright with that. Just wanted to make sure that its not something we missed.